src/ResourceManager/PowerBIEmbedded/Commands.PowerBI/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.Reflection;
```

Comment:
@selasagi this file is missing a license header

Reply:
added